modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -34,6 +35,112 @@ admin_manual:release_notes.adoc
+
+This new server-side capability needs changes in the ownCloud Clients to make use of it. At the time of the ownCloud Server 10.9 release, client implementations for iOS, Android and Desktop are in the making but not yet available. The implementations will follow soon with the next client releases.
+
+TIP: If you use the currently existing versions of the ownCloud Clients, you will not yet experience performance gains.
+
+=== Manual File Locking: Ability to Define User Groups that can Unlock Files
+
+Since version 10.5, xref:file-locking-in-the-web-interface[ownCloud supports Manual File Locking] to prevent concurrent changes of files by different users. The feature works in a way that it only allows the users who locked a file to unlock it again (_exclusive file locking_) and it provides a configurable timeout for file locks.
+
+Version 10.9 introduces a new sub-feature that allows administrators to define groups of users that are entitled to unlock locked files, independent of whether they have set the lock initially or not (https://github.com/owncloud/core/pull/38222[#38222]). Users in such groups can regain control over files, e.g., if a lock owner is unavailable and a long timeout for file locks is used. Administrators can find the configuration in the _Manual File Locking_ section of the _General_ admin settings.
+
+=== File Versions: Store and Display the Author of File Versions
+
+TBC
+
+=== Restrict Public Link Sharing to Certain Groups
+
+In certain scenarios it is not desired that all users of an ownCloud installation can create public links and share data publicly, respectively. To cover for this, ownCloud Server 10.9 comes with a new configuration option for administrators that allows restricting the creation of public links to users in certain groups (https://github.com/owncloud/core/pull/38980[#38980]).
+
+TIP: Existing public links will continue to work after introducing a restriction policy. The policy only applies to the creation of new public links.
+
+=== Ability to move a user's home folder
+
+In some cases it is necessary to manage storage locations of users, e.g., if you want to distribute them across multiple storages. To facilitate this process, Server 10.9 introduces a new _occ_ command (`occ user:move-home`) that allows moving user home folders to a new location (https://github.com/owncloud/core/pull/39136[#39136]). The command takes care of moving the physical data and takes care of updating all other related information.
+Administrators can find more information in the LINK[ownCloud documentation].
+
+=== Updated App Versions
+
+Find below a list of updated apps in comparison with the 10.8 complete bundle. More information on the changes can be found in the respective changelogs on ownCloud Marketplace.
+
+* https://marketplace.owncloud.com/apps/web[Web] 4.x.x
+* Windows Network Drive 2.1.0
+* TBC
```

Comment:
* The following app updates are included in 10.9.0beta1:
   - files_mediaviewer : v1.0.5
   - files_pdfviewer : v1.0.1
   - files_texteditor : v2.4.1
   - guests : v0.9.3
   - oauth2 : v0.5.2
   - openidconnect : v2.1.0
   - web : v4.5.0
   - windows_network_drive : v2.1.0
   - wopi : v1.5.1